account_product_move/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Create extra Journal entries and items when posting out-invoices, that depend on the products in the invoice lines.
```

Comment:
I feel this is a more clear description:

```
This module creates extra Journal Entries for outgoing invoices, depending on the products sold in the invoice lines. To specify the products for which this should happen, the user creates Account Product Move records.
```

Do you agree?

What I'm also missing here is a backstory about the use case in which this can be beneficial, illustrated by an example, along the lines of:

```
For example, a user could decide to create an account product move record with lines of 10 cents for a cone, and 15 cents for ice cold delivery, on an Ice Cream product. This would then lead to extra journal entries of 10 and 15 cents being created, signifying....
```

When you finish the complete text, could you then concatenate it with the USAGE text and put it on the description of the PR on top, so that it's clear and easy to see what's being worked on here?

Reply:
@thomaspaulb yes you are right about this, definitely should be clearer. I will add way more text when the requirements are completely clear to me, because at this very moment, they are still in variable state.

Reply:
Yea i guess that starts with formulating the clients use case, maybe it ends up another solution is better. What does the extra move signify?

Reply:
extra_move = a new journal_entry, created per product in invoice.invoice line, for account_product_move records configured for that product.

 The journal_items of this journal_entry will be the account_product_move_lines of that account_product_move record.

This is the plan for the time being.

Reply:
@ntsirintanis Thats a technical description of what it is, but what we need is an odoo-functional description of what it signifies in the ledger, in the language of the use case.

Reply:
@lfreeke this one's for you.